_posts/2021-05-22-report-arms-spring2021.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title:  "Report: ARMS - reducing robot simulation code by 92%"
+author: "Sean O'Connor"
+categories: [ "robotics", "simulation", "gazebo", "sdf", "javascript"  ]
+image: assets/images/arms1/robot_image.png
+---
+*A concise and intuitive alternative for writing verbose robotics simulation languages, built with browser JavaScript.  For more, check out my [blog](https://oapostrophe.github.io/) and [GitHub](https:/github.com/oapostrophe).*
+
+## Project Background
+
+Let's say you're designing a robot.  This robot will be used to go over rough terrain as part of post-disaster search-and-rescue missions, so it's important that it be both tall enough to clear uneven ground and stable enough not to topple over on bumps.  You want to know: what height and width will offer the optimal combination of ground clearance and stability?
+
+One possible way to find out might be to build several different robots of varying dimensions and test how they perform.  But building each physical robot is expensive and time-consuming, not to mention that there might be a lot of feasible height-weight combinations.  So you also want to know: is there a way we can test different builds, without the cost of physically building every configuration we want to test?
+
+![Selife taken by a robot from the ARCs lab during its construction](../assets/2021-05-22-report-arms-spring2021/robot_selfie.png)  
```

Comment:
As you mentioned, I would prefer the image path to be `/assests/...`

How did you test it out? It probably won't work if you just open the markdown file in your browser without using a local server.

Reply:
I just had GitHub generate a pages website based on the branch in my fork.  I also tried looking at the local Markdown preview.  I changed the paths to be /assets/... for now.